mvlearn/ajive/ajive.py

```diff
@@ -0,0 +1,776 @@
+                else:
+                    U, D, V = svd_wrapper(X_orthog, rank)
+            
+            # projecting X columns onto orthogonal complement of joint_scores
+
+            if self.store_full:
+                if rank == 0:
+                    I = np.zeros(shape=blocks[bn].shape)
+                else:
+                    I = np.array(np.dot(U, np.dot(np.diag(D), V.T)))
+            else:
+                I = None  # Kill I matrix to save memory
+
+            block_specific[bn]['individual'] = {'full': I,
+                                            'scores': U,
+                                            'svals': D,
+                                            'loadings': V,
+                                            'rank': rank}
+
+            # Getting the noise matrix, E
+            if self.store_full and not issparse(X):
+                E = X - (J + I)
+            else:
+                E = None
+            block_specific[bn]['noise'] = E
+
+        # save view specific estimates
+        self.blocks = {}
+
+        # Stores info for easy information checking
+        for bn in block_specific.keys():
+            self.blocks[bn] = \
+            ViewSpecificResults(joint=block_specific[bn]['joint'],
+                                                   individual=block_specific\
+                                                   [bn]['individual'],
+                                                   noise=block_specific[bn]\
+                                                   ['noise'],
+                                                   block_name=bn,
+                                                   obs_names=obs_names,
+                                                   var_names=var_names[bn],
+                                                   m=self.centers_[bn],
+                                                   shape=blocks[bn].shape)
+
+        return self
+
+    @property
+    def is_fit(self):
+        if hasattr(self, "blocks"):
+            return True
+        else:
+            return False
+
+    @property
+    def block_names(self):
+        if self.is_fit:
+            return list(self.blocks.keys())
+        else:
+            return None
+
+    def save(self, fpath, compress=9):
+        dump(self, fpath, compress=compress)
+
+    @classmethod
+    def load(cls, fpath):
+        return load(fpath)
+
+    def get_full_block_estimates(self):
+        """
+
+        Output
+        ------
+        full: dict of dict of np.arrays
+        The joint, individual, and noise full estimates for each block.
+
+        """
+        full = {}
+        for bn in self.block_names:
+            full[bn] = {'joint': self.blocks[bn].joint.full_,
+                        'individual': self.blocks[bn].individual.full_,
+                        'noise': self.blocks[bn].noise_}
+
+        return full
+
+    def results_dict(self):
+        """
+        Returns all estimates as a dicts.
+
```

Comment:
If this is a method that users are allowed to call (not just a private method for within the class), then it needs complete documentation. Same goes for all methods that aren't private